pkg/plugins/platform/local.go

```diff
@@ -80,56 +79,44 @@ func (l *LocalApp) FindContainerByType(containerType string) (docker.APIContaine
 	return dockerutil.FindContainerByLabels(labels)
 }
 
-// Describe returns a string which provides detailed information on services associated with the running site.
-func (l *LocalApp) Describe() (string, error) {
-	maxWidth := uint(200)
-	var output string
-	siteStatus := l.SiteStatus()
+// Describe returns a map which provides detailed information on services associated with the running site.
+func (l *LocalApp) Describe() (map[string]interface{}, error) {
 
-	// Do not show any describe output if we can't find the site.
-	if siteStatus == SiteNotFound {
-		return "", fmt.Errorf("no site found. have you run `ddev start`?")
-	}
-	appTable := CreateAppTable()
+	shortRoot := RenderHomeRootedDir(l.AppRoot())
+	appDesc := make(map[string]interface{})
```

Comment:
This would feel a lot better if were a real type rather than `map[string]interface`. The same struct could be used (or we could use a trimmed down version and rely on composition) for the `ddev list` values.